Workade/Extentions/Color+.swift

```diff
@@ -6,3 +6,31 @@
 //
 
 import Foundation
+import UIKit
+
+// MARK: Color
+// self.view.backgroundColor = .theme.primary
+
+extension UIColor {
+    static let theme = ColorTheme()
+    
+    static func rgb(_ rgbValue: Int) -> UIColor! {
+        return UIColor(
+            red: CGFloat((Float((rgbValue & 0xff0000) >> 16)) / 255.0),
+            green: CGFloat((Float((rgbValue & 0x00ff00) >> 8)) / 255.0),
+            blue: CGFloat((Float((rgbValue & 0x0000ff) >> 0)) / 255.0),
+            alpha: 1.0)
+    }
+}
+
+struct ColorTheme {
+    let primary = UIColor(named: "Primary")!
+    let secondary = UIColor(named: "Secondary")!
+    let tertiary = UIColor(named: "Tertiary")!
+    let quaternary = UIColor(named: "Quaternary")!
+    
+    let background = UIColor(named: "Background")!
+    let labelBackground = UIColor(named: "LabelBackground")!
+    let groupedBackground = UIColor(named: "GroupedBackground")!
+    let subGroupedBackground = UIColor(named: "SubGroupedBackground")!
+}
```

Comment:
```suggestion
struct ColorTheme {
    let primary = UIColor(named: "Primary") ?? UIColor(.black)
    let secondary = UIColor(named: "Secondary") ?? UIColor(.black)
    let tertiary = UIColor(named: "Tertiary") ?? UIColor(.black)
    let quaternary = UIColor(named: "Quaternary") ?? UIColor(.black)
    
    let background = UIColor(named: "Background") ?? UIColor(.black)
    let labelBackground = UIColor(named: "LabelBackground") ?? UIColor(.black)
    let groupedBackground = UIColor(named: "GroupedBackground") ?? UIColor(.black)
    let subGroupedBackground = UIColor(named: "SubGroupedBackground") ?? UIColor(.black)
}
```
[P1] 불필요한 강제 언래핑은 피하는게 좋고 만약 색이 깨진다면 한 눈에 알아보기가 쉽습니다!

Reply:
리뷰 주신대로 반영했습니다! 감사합니다!